src/directLine.test.ts

```diff
@@ -61,3 +65,223 @@ describe("#commonHeaders", () => {
         });
     })
 });
+
+describe("MockSuite", () => {
+
+    const lazyConcat = <T>(items: Iterable<Observable<T>>): Observable<T> =>
+        new Observable<T>(subscriber => {
+            const iterator = items[Symbol.iterator]();
+            let inner: Subscription | undefined;
```

Comment:
Is this one supported in TypeScript?

```suggestion
            let inner?: Subscription;
            // or
            let inner: Subscription?;
```

Reply:
? is mostly used to mark option parameters to a function.